modules/frontend/metrics_query_range_sharder.go

```diff
@@ -438,7 +439,10 @@ func (s *queryRangeSharder) generatorRequest(searchReq tempopb.QueryRangeRequest
 
 	searchReq.QueryMode = querier.QueryModeRecent
 
-	return s.toUpstreamRequest(parent.Context(), searchReq, parent, tenantID)
+	req := s.toUpstreamRequest(parent.Context(), searchReq, parent, tenantID)
+	req.Header.Set(api.HeaderAccept, api.HeaderAcceptProtobuf)
```

Comment:
should we add this proto header in `prepareRequestForQueriers()`? 

this feels like a few changes away from just making this the default relationship between the queriers and frontend.

also, fantastic find! agree with the analysis this is largely cropping up due to the streaming frontend refactor.

Reply:
Yes, when we make this universal that is the ideal place to add the header.  And on the querier side the `writeFormattedContentForRequest(w, r, resp)` is also easy to incorporate.  I'm on the fence to do it in this PR, since splitting out the remaining work would be a nice opportunity to get others get involved.  But happy to do it if that's not needed.

Reply:
my .02: just do them all for consistency and simplicity

i don't disagree with your thoughts though if you decide to just merge